README.md

```diff
@@ -33,9 +33,20 @@ Afterwards, install Keptn CLI:
 curl -sL https://get.keptn.sh | bash
 ```
 
-And install Keptn (Note: remove `--use-case=continuous-delivery` in case you don't want to install `helm-service` and `jmeter-service`):
+And install Keptn usint Helm:
 ```console
-keptn install --use-case=continuous-delivery
+helm repo add keptn https://charts.keptn.sh && helm repo update
+helm install keptn keptn/keptn \
+-n keptn --create-namespace \
+--wait \
+--set=control-plane.apiGatewayNginx.type=LoadBalancer
```

Comment:
```suggestion
--set=apiGatewayNginx.type=LoadBalancer
```

Reply:
took it from here: should we change there as well?https://keptn.sh/docs/quickstart/

Reply:
aha yes, i will update that, thanks!